aws/resource_aws_efs_access_point_test.go

```diff
@@ -0,0 +1,515 @@
+package aws
+
+import (
+	"fmt"
+	"log"
+	"regexp"
+	"testing"
+	"time"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/efs"
+
+	multierror "github.com/hashicorp/go-multierror"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/terraform"
+)
+
+func init() {
+	resource.AddTestSweepers("aws_efs_access_point", &resource.Sweeper{
+		Name: "aws_efs_access_point",
+		F:    testSweepEfsAccessPoints,
+	})
+}
+
+func testSweepEfsAccessPoints(region string) error {
+	client, err := sharedClientForRegion(region)
+	if err != nil {
+		return fmt.Errorf("error getting client: %s", err)
+	}
+	conn := client.(*AWSClient).efsconn
+
+	var errors error
+	input := &efs.DescribeFileSystemsInput{}
+	err = conn.DescribeFileSystemsPages(input, func(page *efs.DescribeFileSystemsOutput, lastPage bool) bool {
+		for _, filesystem := range page.FileSystems {
+			id := aws.StringValue(filesystem.FileSystemId)
+			log.Printf("[INFO] Deleting access points for EFS File System: %s", id)
+
+			var errors error
+			input := &efs.DescribeAccessPointsInput{
+				FileSystemId: filesystem.FileSystemId,
+			}
```

Comment:
Can we get away from needing the wrapping `DescribeFileSystems` handling by omitting `FileSystemId`?